worlds/tits_the_3rd/client/client.py

```diff
@@ -253,6 +269,21 @@ async def tits_the_3rd_watcher(ctx: TitsThe3rdContext):
             logger.info(f"Game Start Dectected. Setting up AP Verification Hook")
             ctx.game_interface.write_world_player_identifier(ctx.world_player_identifier)
             logger.info(f"AP Verification Hook Set. You can now save and resume the game for this AP Seed")
+            for character_item in ctx.items_received[:2]:  # The first 2 items are guarantee to be characters
```

Comment:
I don't think this logic holds up. There's a universal setting on every AP yaml which lets you specify starting items. Let's check the entire array here for character items.

Reply:
Actually, I wonder how this plays into it. Maybe these get sent as normal items. We should probably test the case where someone specifies a 3rd or 4th or 5th character to start with.

Similarly, there's the case where we don't start the game until we've already received more characters.

Reply:
This is purely for the initial characters. If you receive more characters and start a new game they will be sent like a normal item. This is to tell the game give me these characters so I have someone to control at the beginning